### PR TITLE
Allow parameter data to be treated as commands for the ssd1331

### LINKED ISSUE #1750: Allow displayio to treat all SPI parameters as commands
The SSD1331 likes everything sent as commands. I could format the INIT data as all commands, but that gets lengthy, so it would be nice to just pass a parameter to allow this to happen automatically. Additionally all of the boundary parameters need to be treated as commands too.

### PR DESCRIPTION
**Pull request by makermelissa**
This fixes #1750

### CODE REVIEW

File: shared-module/displayio/Display.c [hunk 1]
```diff
@@ -82,7 +83,13 @@ void common_hal_displayio_display_construct(displayio_display_obj_t* self,
         data_size &= ~DELAY;
         uint8_t *data = cmd + 2;
         self->send(self->bus, true, cmd, 1);
-        self->send(self->bus, false, data, data_size);
+        if (self->data_as_commands) {
+            for (uint32_t j=0; j < data_size; j++) {
```
Comment (tannewt) [inline shared-module/displayio/Display.c]: Why is this in a for loop rather that one call for all of the data? Is it because CS needs to be toggled?

Comment (tannewt) [reply shared-module/displayio/Display.c]: (Please add the explanation as a comment if it is needed.) Thanks!

Comment (makermelissa) [reply shared-module/displayio/Display.c]: Yes, exactly.

Comment (makermelissa) [reply shared-module/displayio/Display.c]: Done.

**Comment by makermelissa** [comment]
Looks like the extra bytes are causing the same issue with #1733 to pop up again and travis is failing. What would be the best strategy to fix this?

**Comment by dhalbert** [comment]
In `ports/atmel-samd/boards/hallowing_m0_express/mpconfigboard.mk`, lower `CFLAGS_INLINE_LIMIT` to 50, and/or turn off some features (e.g., `CIRCUITPY_SAMD = 0`)

**Comment by dhalbert** [comment]
Do a local test with the `pl` build to vet the change.

**Comment by makermelissa** [comment]
Thanks Dan. Adjusted the limit and local PL build succeeded. Trying again.

**Pull request merged**